session_items.py

```diff
@@ -67,3 +67,16 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def remove_item(item):
+    """
+    Removes an existing item in the session. If no existing item matches the ID of the specified item, nothing is removed.
+
+    Args:
+        item: The item to remove.
+    """
+    existing_items = get_items()
```

Comment:
How would you achieve the same result using a list comprehension?

Reply:
as in something like `session['items'] = [listItem for listItem in getItems() if listItem != item]` as the whole method?

Reply:
Yes :)